.vib/flux/runtime-parameters.yaml

```diff
@@ -90,7 +90,7 @@ extraDeploy:
         "helm.sh/hook": post-install
     spec:
       interval: 5m
-      url: https://github.com/javsalgar/podinfo
+      url: https://github.com/mdhont/podinfo
```

Comment:
Is this intended?

Reply:
Yes, it's a new fork from the one upstream, as the current fork has changes that do not work here. I can also use the upstream url.